src/file_operations.js

```diff
@@ -28,7 +28,15 @@ const makeUidInfo = courseData => {
   // or other places which may need a little bit of context on external items
 
   const types = {
```

Comment:
This is a bit of a nitpick, but the name of this function and the name of the constant `types` is a bit confusing.  The function returns a lookup, so why not call the function `makeUidInfoLookup` or something along those lines?  As far as `types`, it looks like originally these objects only contained a `type` property, but now they contain a lot more than that.  Something a bit more descriptive like `uidInfoObjects` might make more sense.

Reply:
Sounds good, just committed the change